.buildkite/commands/run-unit-tests.sh

```diff
@@ -36,6 +36,11 @@ else
   echo "The Unit Tests, which ran inside the '🧪 Testing' section above in the logs, have failed."
   echo "For more details about the failed tests, check the Buildkite annotation, the logs under the '🧪 Testing' section and the \`.xcresult\` and test reports in Buildkite artifacts."
 fi
-annotate_test_failures "fastlane/test_output/WooCommerce.xml"
+
+if [ "$BUILDKITE_BRANCH" = "trunk" ]; then
+    annotate_test_failures "fastlane/test_output/WooCommerce.xml" --slack "build-and-ship"
```

Comment:
Open question: I wonder if you wouldn't want to get Slack notifications for any `release/*` branch too? 🤔

Reply:
That's a good question. Thinking about faster feedback, it makes sense.

Reply:
while the original scope was only to turn this on for trunk, yeah i agree it's probably worth extending this to include `release/*` branches too, especially since we saw at least one release build recently that started failing because of tests 😬 

i updated the condition to include `release/*` in https://github.com/woocommerce/woocommerce-ios/pull/9853/commits/06c13dd64d3a33cf07681554c914524b7257bcb4 and tested that using this [draft PR](https://github.com/woocommerce/woocommerce-ios/pull/9856) (used `test/` instead of `release/` to test the condition).